pylib/anki/collection.py

```diff
@@ -303,6 +303,12 @@ def beforeUpload(self) -> None:
     # Object creation helpers
     ##########################################################################
 
+    def siblings(self, cids):
```

Comment:
While not a bad place to put this, you could just use the existing browser .selectedNotesAsCards() method instead

Reply:
Oh right. I never saw it. Thanks